src/test/java/fi/aalto/cs/apluscourses/integration/ApiTest.java

```diff
@@ -14,6 +16,24 @@
   @ClassRule
   public static final EnvironmentChecker checker = new EnvironmentChecker("CI");
 
+  @Test
+  public void getSubmissionsResultsReturns() throws MalformedURLException {
```

Comment:
there is no data unless https://github.com/Aalto-LeTech/intellij-plugin/issues/234 is ready